jf_agent/agent_logging.py

```diff
@@ -189,7 +199,7 @@ def handle(self, record: Union[logging.LogRecord, int]) -> None:
 
 @dataclass
 class AgentLoggingConfig:
-    level: str
+    level: int
     datefmt: str
     handlers: List[str]
     listener: QueueListener
```

Comment:
Updating this since logging level is a int rather than a string. Also, we might want to reconsider if this dataclass is actually necessary